codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithyjava/nameresolver/Native.java

```diff
@@ -378,4 +372,8 @@ public ClassName baseErrorForService() {
       "Use `ClassName.get(RuntimeException.class)` instead."
     );
   }
+
+  protected ClassName classNameForAwsSdkShape(final Shape shape) {
+    throw new UnsupportedOperationException();
```

Comment:
Can we add a message here to explain why it's unsupported?

Reply:
Did you one better and made `Native` abstract so this could be abstract too. :)

Reply:
Scratch that, the `Native` class is used directly for non SDK-style projects as well. I reverted and added an exception message instead. The better change would be to have an intermediate base class in between `Native` and `AwsSdkNativeV1/2`, but that looks like a substantial refactoring.